### ISSUE TITLE
linked image accessibility

### ISSUE THREAD

**Issue by eshellman** [gutenbergtools/ebookmaker#278]
many books have links to raw images, anchored with a small view of the same image. 
e.g. `<a href="image.png"><img src="image.small.png" alt=""></a>`

This is usually intended to provide a full-size version of the image when a smaller version is clicked. When this occurs, an assistive screen reader will announce the link using the alt-text of the anchor image. Not sure what to do wrt accessibility.

My Guess: 
If image has alt text, do nothing?
If image has no alt text, use aria-hidden?
